balancer/weightedroundrobin/balancer.go

```diff
@@ -295,6 +296,7 @@ func (b *wrrBalancer) regeneratePicker() {
 	}
 
 	p := &picker{
+		idx:      grpcrand.Uint32(),
```

Comment:
Minor (will rarely rarely happen): what happens if this generates something close to max uint 32, and then overflows. Does it just wrap back to start?